### PR TITLE
ECO-4901 Define colors in variables.less throughout

### PR DESCRIPTION
**Pull request by Raulkg**
- isolated variables in the variables file

### CODE REVIEW

File: web/less/variables.less [hunk 1]
```diff
@@ -71,6 +71,87 @@
 @grey5: #686868;
 @grey6: #31313d;
 
+@VeryLightGrey: #CFCFCF;
+@lightGrey: #404040;
+@lightGrey1: #d7d7d7;
+@lightGrey2: #E8E8E8;
+@lightGrey3: #e7e7e7;
+
+@BaliHaiGrey: #8E99A5;
```
Comment (jeffswartz) [inline web/less/variables.less:80]: For consistency, shouldn't these all use lower camel case, e.g. `@baliHaiGrey`?

**Review by jeffswartz — CHANGES_REQUESTED** [review]
There are still some colors in the CSS that don't use variable names. Search for `\box-shadow:.*?#\w+\` (regex) or `'color: #'`. For example, /web/css/buttons.css includes the following:
```
.tb.btn.btn-primary {
  background-color: #0099cc;
  box-shadow: 0 0 0 1px #0099cc;
  color: #fff;
}
```

**Comment by Raulkg** [comment]
> color: #

the only place i see the colors without variables are in the below files:
1) min.less (obsolete as they are being removed in other Ticket)
2) bubbles.less  (Already used as variables scoped to this file)
3) landing.less (L247: background-color: #00000099;)  (Invalid hex need your review here @jeffswartz )

File: web/less/variables.less (hunk 1 above)
Comment (Raulkg) [reply web/less/variables.less]: updated in new commit 
`b7ea82d -> 8730cfa`

**Review by jeffswartz — APPROVED** [review]

File: web/less/variables.less (hunk 1 above)
Comment (jeffswartz) [reply web/less/variables.less]: Never mind. I was looking at generated files.

**Pull request merged**